airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -97,14 +97,14 @@ def get_iam_execution_role(self) -> Dict:
             self.log.error("Failed to create aws glue job, error: %s", general_error)
             raise
 
-    def initialize_job(self, script_arguments: Optional[List] = None) -> Dict[str, str]:
+    def initialize_job(self, script_arguments: Optional[dict] = None) -> Dict[str, str]:
```

Comment:
`script_arguments` is passed to `Glue.Client.start_job_run` as `Arguments` whose type is `dict`.
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue.html#Glue.Client.start_job_run